db/seeds.rb

```diff
@@ -44,6 +44,8 @@
 TOS_TEXT = File.read(File.join(DEFAULT_MSGS, 'tos_text'))
 UPCOMING_CHECKIN_RES_EMAIL = File.read(File.join(DEFAULT_MSGS,
                                                  'upcoming_checkin_email'))
+UPCOMING_CHECKOUT_RES_EMAIL = File.read(File.join(DEFAULT_MSGS,
```

Comment:
You need to also assign this to the generated `AppConfig` object below (~L394), otherwise it isn't used.